metapool/util.py

```diff
@@ -165,8 +165,8 @@ def _check_for_missing_df_ids(subset_df, superset_df, id_col,
                          f'{", ".join(missing_vals)}')
 
 
-def get_set_fp(set_fbase, set_id):
-    return f"{set_fbase}{SET_SUFFIX}_{set_id}.txt"
+def get_set_fp(set_fbase, set_id, extension="txt"):
```

Comment:
I can see in the notebooks that this can be txt or csv; why allow that vs. making all the files the same extension? I don't have a preference on which one to use, just that supporting both might not be ideal.

Reply:
I am using the same method to save the set-specific samplesheets, which are csvs, and the set-specific plate map files (which are txts).  Personally I would prefer that the plate map files also be csvs, but I'm not sure I can take on changing that at the moment since it also impacts the matrix tube (and maybe metaT) notebook.

Reply:
Got it and agree that this will be a larger refactor, opened an issue about it.